service/maintut.go

```diff
@@ -1,12 +1,13 @@
 package main
 
 import (
+	"github.com/Harkishen-Singh/Jarvis-personal-assistant/service/config"
 	"github.com/Harkishen-Singh/Jarvis-personal-assistant/service/utils"
 )
 
-
 func main() {
 	// utils.LoggerWarn("starting Jarvis service ...")
 	// initiate service
+	config.Init()
```

Comment:
After changing the function name to `init()`, there won't be required of calling and initialising the functions again.